vendir.yml

```diff
@@ -0,0 +1,16 @@
+apiVersion: vendir.k14s.io/v1alpha1
+kind: Config
+directories:
+  - path: vendor
+    contents:
+      - path: cilium
+        git:
+          url: https://github.com/giantswarm/cilium
+          ref: apply-gs-customizations
```

Comment:
We are currently targeting a work-in-progress branch. Later I'll change it to use a stable branch from our fork.